_data/talks.yml

```diff
@@ -86,6 +88,26 @@
   presenter1_profile: https://layer5.io/community/members/lee-calcote
   new_window: true
 
+- title: Measuring Service Mesh Performance 101
+  video: https://www.youtube.com/watch?v=hzYy52A1VTw
+  embeded_video: https://www.youtube.com/embed/hzYy52A1VTw
+  presentation: TBA
```

Comment:
Until @navendu-pottekkat provides a link, this PR should not be merged. 

Either remove the entry for that talk or await his update.

Reply:
Yess, I will wait for an update from him 👀

Reply:
@adithyaakrishna no update thus far. I assume that there won't be. Will you refactor based on the lack of input?

Reply:
Got an Update from him (had missed his notification) and added link to the slide as well 👀